<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>memberPurchases</class>
 <widget class="QWidget" name="memberPurchases">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Member Purchases</string>
  </property>
  <property name="styleSheet">
   <string notr="true">\QPushButton{
	background-color: rgb(46, 52, 54);
	border: none;
	color: rgb(255, 255, 255);
	border-left: 1px solid rgb(0, 0, 0);
	border-right: 1px solid rgb(0, 0, 0);
	border-bottom: 3px solid rgb(0, 0, 0);
}

QPushButton:hover{
	background-color: rgb(59, 59, 59);
	border: none;
	color: rgb(255, 255, 255);
	border-left: 1px solid rgb(29, 29, 29);
	border-right: 1px solid rgb(29, 29, 29);
	border-bottom: 3px solid rgb(29, 29, 29);
}

QPushButton:pressed{
	background-color: rgb(46, 52, 54);
	border: none;
	color: rgb(255, 255, 255);
	border-left: 1px solid rgb(0, 0, 0);
	border-right: 1px solid rgb(0, 0, 0);
	border-top: 3px solid rgb(0, 0, 0);
}

QDialog{
	background-color: rgb(85, 87, 83);
}

QWidget{
	background-color: rgb(85, 87, 83);
}

QMainWindow{
	background-color: rgb(85, 87, 83);
}

QLabel{
	color: rgb(255, 255, 255);
	
}

QLineEdit{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 10px;
	color: #FFF;
	background-color:rgb(34, 36, 44);
}

QLineEdit:hover{
	border: 2px solid rgb(48, 50, 62);
}

QLineEdit:focus{
	border: 2px solid rgb(85, 170, 255);
	background-color:rgb(43, 45, 56);
}


/*--------------------------------------------------------------------------------------------*/

 QScrollBar:vertical {
	border: none;
    background: rgb(45, 45, 68);
    width: 14px;
    margin: 15px 0 15px 0;
	border-radius: 0px;
 }

/*  HANDLE BAR VERTICAL */
QScrollBar::handle:vertical {	
	background-color: rgb(80, 80, 122);
	min-height: 30px;
	border-radius: 7px;
}
QScrollBar::handle:vertical:hover{	
	background-color: rgb(255, 0, 127);
}
QScrollBar::handle:vertical:pressed {	
	background-color: rgb(185, 0, 92);
}

/* BTN TOP - SCROLLBAR */
QScrollBar::sub-line:vertical {
	border: none;
	background-color: rgb(59, 59, 90);
	height: 15px;
	border-top-left-radius: 7px;
	border-top-right-radius: 7px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical:hover {	
	background-color: rgb(255, 0, 127);
}
QScrollBar::sub-line:vertical:pressed {	
	background-color: rgb(185, 0, 92);
}

/* BTN BOTTOM - SCROLLBAR */
QScrollBar::add-line:vertical {
	border: none;
	background-color: rgb(59, 59, 90);
	height: 15px;
	border-bottom-left-radius: 7px;
	border-bottom-right-radius: 7px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
}
QScrollBar::add-line:vertical:hover {	
	background-color: rgb(255, 0, 127);
}
QScrollBar::add-line:vertical:pressed {	
	background-color: rgb(185, 0, 92);
}

/* RESET ARROW */
QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
	background: none;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
	background: none;
}


QTableView{
	background-color: rgb(186, 189, 182);
}



QScrollBar:horizontal {
    border: none;
    background: rgb(45, 45, 68);
    width: 14px;
    margin: 15px 0 15px 0;
	border-radius: 0px;
}
QScrollBar::handle:horizontal {
    background-color: rgb(80, 80, 122);
	min-height: 30px;
	border-radius: 7px;
}
QScrollBar::add-line:horizontal {
    background-color: rgb(255, 0, 127);
}

QScrollBar::sub-line:horizontal {
    background-color: rgb(185, 0, 92);
}
QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal
{
    background: none;
}
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal
{
    background: none;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="memberidlabel">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Member ID: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="memberIDLineEdit">
           <property name="placeholderText">
            <string>6 Digits</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="idSearchButton">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Member Name: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="memberNameLineEdit">
           <property name="placeholderText">
            <string>John Smith</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nameSearchButton">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="totalLabel">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Total Purchases +tax: </string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Exit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="DisplayAllPushButton">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Display All</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
